query_execution/CMakeLists.txt

```diff
@@ -235,36 +241,6 @@ if (ENABLE_DISTRIBUTED)
   add_test(BlockLocator_unittest BlockLocator_unittest)
 endif()
 
-add_executable(Foreman_unittest
-  "${CMAKE_CURRENT_SOURCE_DIR}/tests/Foreman_unittest.cpp")
```

Comment:
@hbdeshmukh Since we removed this test, please also remove its source file. Thanks!